janus_server/src/bin/janus_cli.rs

```diff
@@ -282,7 +284,7 @@ struct KubernetesSecretOptions {
         env = "DATASTORE_KEYS_SECRET_KEY",
         num_args = 1,
         help = "Key into data of datastore key Kubernetes secret",
-        default_value = "datastore_key"
+        default_value = "datastore_keys"
```

Comment:
I think this default value is incorrect, see https://github.com/divviup/janus-ops/blob/148ca62f37758275e70cbe030fa5fb5babffa967/terraform/main_gcp/main.tf#L221